components/molecule/breadcrumb/src/settings.scss

```diff
@@ -9,3 +9,4 @@ $c-breadcrumb: $c-text-base !default;
 $m-breadcrumb-items: $m-s !default;
 $size-breadcrumb-icon: 18px !default;
 $td-breadcrumb-link-hover: none !default;
+$fz-breadcrumb-item: $fz-m !default;
```

Comment:
this isn't the value it had before, right? this would mean that it won't be backward compatible

Reply:
It wasn't set before. In the sui demo it was taking the body fz, but if you remove it it goes to the `$fz-b` that is `$fz-m`